.github/workflows/cd.yml

```
@@ -28,44 +39,79 @@ jobs:
       fail-fast: false
       matrix:
         platform: [iOS, macOS]
-        uploadto: [app-store, ftp]
     runs-on: macos-13
 
     steps:
+
     - name: Checkout code
       uses: actions/checkout@v4
 
-    - name: Set iOS extra xcode params
-      if: matrix.platform == 'iOS'
-      run: echo "EXTRA_XCODEBUILD=-sdk iphoneos ${{ env.APPLE_AUTH_PARAMS }}" >> $GITHUB_ENV
+    - name: Install python dependencies
+      run: pip install pyyaml
```

```suggestion
      run: pip install pyyaml==6.0.1
```
You don't want this to break because of a dep update